server/services/store/sqlstore/blocks.go

```diff
@@ -182,7 +182,7 @@ func (s *SQLStore) getSubTree3(db sq.BaseRunner, c store.Container, blockID stri
 		Join(s.tablePrefix + "blocks" + " as l3 on l3.parent_id = l2.id or l3.id = l2.id").
 		Where(sq.Eq{"l1.id": blockID}).
 		Where(sq.Eq{"COALESCE(l3.workspace_id, '0')": c.WorkspaceID}).
-		OrderBy("l3.insert_at")
+		OrderBy("l3.id")
```

Comment:
This is not strictly the same, why are we changing this?, maybe would have fixed it with ordering by `insert_at` and `id`?

Reply:
Order preserved via https://github.com/mattermost/focalboard/pull/2000/commits/dda97326628ba77882fbae6d8232e3a3a1c346e4